app/shared/buttons/SVGIconChooser.js

```diff
@@ -0,0 +1,86 @@
+import React, { useRef, useState } from 'react';
```

Comment:
does this deserve to be in the shared folder?

Reply:
missing 'use client' at the top

Reply:
See this comment re: shared folder usage
https://github.com/thegoodparty/gp-webapp/pull/122/files#r1529744815